Remotion/ObjectBinding/Web.Test/IndividualControlTests/BocListUserControl.ascx.cs

```diff
@@ -392,6 +400,186 @@ private void ChildrenListRemoveRowsButton_Click (object sender, EventArgs e)
+                // First child failure (visible)
+                new ObjectValidationFailure(
+                    firstChild,
+                    "Bad first child",
+                    "Localized bad first child"),
+                new PropertyValidationFailure(
+                    firstChild,
+                    personPartnerProperty,
+                    firstChild.Partner,
+                    "Bad first child.Partner",
+                    "Localized bad first child.Partner"),
+                new ObjectValidationFailure(
+                    firstChild,
+                    new[]
+                    {
+                        new ValidatedProperty(firstChild, personLastNameProperty),
+                    },
+                    "Bad first child name",
+                    "Localized bad first child names"),
+                new ObjectValidationFailure(
+                    firstChild,
+                    new[]
+                    {
+                        new ValidatedProperty(firstChild, personLastNameProperty),
+                        new ValidatedProperty(firstChild, personFullNameProperty)
+                    },
+                    "Bad first child names",
+                    "Localized bad first child names"),
+                // Last child failure (invisible)
+                new ObjectValidationFailure(
+                    lastChild,
+                    "Bad last child",
+                    "Localized bad last child"),
+                new PropertyValidationFailure(
+                    lastChild,
+                    personPartnerProperty,
+                    lastChild.Partner,
+                    "Bad last child.Partner",
+                    "Localized bad last child.Partner"),
+                new ObjectValidationFailure(
+                    lastChild,
+                    new[]
+                    {
+                        new ValidatedProperty(lastChild, personLastNameProperty),
+                    },
+                    "Bad last child name",
+                    "Localized bad last child name"),
+                new ObjectValidationFailure(
+                    lastChild,
+                    new[]
+                    {
+                        new ValidatedProperty(lastChild, personLastNameProperty),
+                        new ValidatedProperty(lastChild, personFullNameProperty)
+                    },
+                    "Bad last child names",
+                    "Localized bad last child names"),
+            }));
+
+    PrepareValidation();
+    FormGridManager.PrepareValidation();
+    DataSourceValidationResultDispatchingValidator.DispatchValidationFailures(validationResult);
+    DataSourceValidationResultDispatchingValidator.Validate();
+
+    UnhandledValidationErrorsLabel.Text = string.Join(
+        Environment.NewLine,
+        validationResult.GetUnhandledValidationFailures().Select(FormatValidationFailure));
+
+    static string FormatValidationFailure (ValidationFailure failure)
```

Comment:
This method doesn't seem to get used?

Reply:
resolved as the usage is visible two lines above ;)